andersen-homework/src/main/java/Main.java

```diff
@@ -0,0 +1,23 @@
+import org.model.Ticket;
+import org.util.BusTicketValidator;
+
+import java.io.IOException;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        BusTicketValidator validator = new BusTicketValidator();
+
+
+        try {
+            // Load tickets from JSON file
+            List<Ticket> tickets = validator.loadTicketsFromFile("tickets.json");
```

Comment:
save file name in separate constant(also could be stored in config file)

Reply:
I have added ticket filename as a configurable property in application.properties

Reply:
check it again teacher please